.buildkite/pipeline.yml

```diff
@@ -8,7 +8,7 @@ steps:
       platform: x86_64.metal
     plugins:
       - docker#v3.0.1:
-          image: "fandree/rust-vmm-dev"
+          image: "rustvmm/dev:v3"
```

Comment:
This should be deleted if we're switching to rust-vmm-ci.

Reply:
I just copied what vm-memory is doing... Is that wrong as well?

Reply:
The vm-memory setup is correct. The vm-memory has a specialized pipeline because it has features and the default buildkite pipeline does not have support for features. You'll notice that the Buildkite pipeline in the vm-memory repository only runs the tests related to the mmap feature: https://github.com/rust-vmm/vm-memory/blob/master/.buildkite/pipeline.linux.yml

For the custom, default tests we can use the existing pipeline.